src/java.base/share/classes/javax/crypto/CryptoAllPermission.java

```diff
@@ -61,19 +61,19 @@ private CryptoAllPermission() {
      * @param p the permission to check against.
      *
      * @return true if the specified permission is an
-     * instance of CryptoPermission.
+     * instance of <code>CryptoPermission</code>.
```

Comment:
Use {@code } for all of the javadoc in this class?

Reply:
using `{@code }`